CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
-project(binaryen C CXX)
 cmake_minimum_required(VERSION 3.1.3)
+project(binaryen LANGUAGES C CXX VERSION 90)
```

Comment:
Will we need to change this each time we update the version? Alternatively, could this have `"unknown"` instead of `90`?

Reply:
Yes we will need to change this when we do releases.. although if we forget it only effects people who don't use git.. and not a way that is particularly problematic.

Reply:
I think it would be better to report "unknown" than possibly a wrong version number, myself. But I don't feel strongly.

Reply:
But that means that anyone who packages binaryen from a source archive alwasy gets an "unknown" version.

I'd rather write a test that the CHANGELOG matches this version..  but I think that might be overkill.